### PR TITLE
Get rid of most of inspection warnings

### PR DESCRIPTION
**Pull request by outcatcher**
Try to get rid of most inspection warning that won't require much extra work

### CODE REVIEW

File: openstack/bms/v2/keypairs/results.go [hunk 1]
```diff
@@ -49,17 +48,3 @@ func ExtractKeyPairs(r pagination.Page) ([]KeyPair, error) {
...
 	return results, err
 }
-
-type keyPairResult struct {
-	golangsdk.Result
-}
-
-// Extract is a method that attempts to interpret any KeyPair resource response
-// as a KeyPair struct.
-func (r keyPairResult) Extract() (*KeyPair, error) {
```
Comment (lego963) [inline openstack/bms/v2/keypairs/results.go:59]: do we use it terraform?

File: openstack/ims/v2/cloudimages/results.go [hunk 2]
```diff
@@ -99,22 +99,6 @@ func (r *Image) UnmarshalJSON(b []byte) error {
...
-
-// Extract will get the Image object out of the commonResult object.
-func (r commonResult) Extract() (*Image, error) {
-	var s Image
-	err := r.ExtractInto(&s)
-	return &s, err
-}
-
-// ExtractInto converts our response data into a volume struct
-func (r commonResult) ExtractInto(v interface{}) error {
```
Comment (lego963) [inline openstack/ims/v2/cloudimages/results.go:114]: redundant code?

File: openstack/networking/v2/extensions/fwaas/firewalls/errors.go [hunk 3]
```diff
@@ -1,11 +1 @@
 package firewalls
-
```
Comment (lego963) [inline openstack/networking/v2/extensions/fwaas/firewalls/errors.go:2]: remove this file

File: openstack/networking/v2/extensions/fwaas/rules/errors.go [hunk 4]
```diff
@@ -1,12 +1 @@
 package rules
-
-import "fmt"
```
Comment (lego963) [inline openstack/networking/v2/extensions/fwaas/rules/errors.go:3]: same as previous

File: openstack/networking/v2/extensions/fwaas_v2/firewall_groups/errors.go [hunk 5]
```diff
@@ -1,11 +1 @@
 package firewall_groups
-
-import "fmt"
```
Comment (lego963) [inline openstack/networking/v2/extensions/fwaas_v2/firewall_groups/errors.go:3]: same

File: openstack/networking/v2/extensions/fwaas_v2/rules/errors.go [hunk 6]
```diff
@@ -1,12 +1 @@
 package rules
-
-import "fmt"
```
Comment (lego963) [inline openstack/networking/v2/extensions/fwaas_v2/rules/errors.go:3]: plus one

**Review by lego963 — CHANGES_REQUESTED** [review]

**Review by anton-sidelnikov — APPROVED** [review]

**Review by lego963 — APPROVED** [review]

**Pull request merged**